Kickstarter-iOS/Views/Controllers/ThanksViewController.swift

```diff
@@ -185,18 +186,22 @@ internal final class ThanksViewController: UIViewController, UICollectionViewDel
   }
 
   fileprivate func showRatingAlert() {
-    self.present(
-      UIAlertController.rating(
-        yesHandler: { [weak self] _ in
-          self?.viewModel.inputs.rateNowButtonTapped()
-        }, remindHandler: { [weak self] _ in
-          self?.viewModel.inputs.rateRemindLaterButtonTapped()
-        }, noHandler: { [weak self] _ in
-          self?.viewModel.inputs.rateNoThanksButtonTapped()
-      }),
-      animated: true,
-      completion: nil
-    )
+    if #available(iOS 10.3, *) {
+      SKStoreReviewController.requestReview()
```

Comment:
There's no input to feed back into our view model. There's no mechanism to know if the alert was displayed, if the user reviewed or canceled, etc.